### ISSUE TITLE
Error regarding missing js-yaml dependency

### ISSUE THREAD

**Issue by frontendphil** [JasonEtco/actions-toolkit#110]
We've just received the 3.0 update in our pipeline. However, actions that use `actions-toolkit` now all fail with the following error: 

```
Error: Cannot find module 'js-yaml'
Require stack:
- /node_modules/actions-toolkit/lib/index.js
- /entrypoint.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:980:15)
    at Function.Module._load (internal/modules/cjs/loader.js:862:27)
    at Module.require (internal/modules/cjs/loader.js:1040:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/node_modules/actions-toolkit/lib/index.js:54:33)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1171:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Module.require (internal/modules/cjs/loader.js:1040:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/node_modules/actions-toolkit/lib/index.js', '/entrypoint.js' ]
}
```

Does that ring a bell?

**Comment by JasonEtco** [comment]
Hey @frontendphil, can you share your code? Js-yaml isn’t used by this library anymore, but if you were using the dependency directly you’ll now need to install it directly.

**Comment by frontendphil** [comment]
That's weird :) The code is rather simple. 

```js
// entrypoint.js
const { Toolkit } = require('actions-toolkit')

const tools = new Toolkit()

const { owner, repo } = tools.context.repo
const { sha } = tools.context
const { status, message, context } = tools.inputs

tools.github.repos
  .createStatus({
    owner,
    repo,
    sha,
    state: status,
    description: message,
    context,
    target_url: `https://www.github.com/${owner}/${repo}/commit/${sha}/checks`,
  })
  .then(() => {
    tools.exit.success()
  })
  .catch(err => {
    tools.log.fatal(err)
    tools.exit.failure()
  })
```



```dockerfile
// Dockerfile
FROM node:slim

COPY package*.json ./

RUN npm ci

COPY entrypoint.js /entrypoint.js

ENTRYPOINT ["node", "/entrypoint.js"]
```

**Comment by JasonEtco** [comment]
Thank you for the info @frontendphil 🙏 you're absolutely right - I published the wrong built code 🤦‍♂ I had some automation in this repo that broke and then forgot to do a manual step.

I've just published 3.0.1, please let me know if that works for you!

**Comment by frontendphil** [comment]
👍 works like a charm now!
